src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -695,6 +698,64 @@ class spell_mage_water_elemental_freeze : public SpellScript
     }
 };
 
+/* 112965 - Finger of Frost
+   228597 - Frostbolt
+   84721  - Frozen Orb */
+class spell_mage_finger_of_frost : public SpellScriptLoader
+{
+public:
+    spell_mage_finger_of_frost() : SpellScriptLoader("spell_mage_finger_of_frost") { }
```

Comment:
drop this part and go with the RegisterSpellScript macro like all other spell scripts do

Reply:
Done and tested. Next ones i will use same, thanks! ;)